specificatie/filter.yaml

```diff
@@ -8,8 +8,12 @@ components:
   schemas:
     Field:
       description: |
+          Hiermee kun je de inhoud van de resource naar behoefte aanpassen door een lijst van paden die verwijzen naar de gewenste velden op te nemen 
```

Comment:
Ik weet niet of dit duidelijk is. Hier wordt over een lijst gesproken terwijl dit het pad is naar een enkele veld is.  Voorstel:

Hiermee kun je het pad van het gewenste veld van een resource opgeven.

Reply:
Aangepast